src/main/java/com/pandora/plugin/ui/FileSearchDialog.kt

```diff
@@ -73,7 +65,7 @@ class FileSearchDialog(project: Project,
 
     private fun createTextComponent(str: String): JComponent {
         val textLabel = JLabel(str)
-        textLabel.ui = MultiLineLabelUI()
+        textLabel.setUI(MultiLineLabelUI())
```

Comment:
The compiler was complaining about reassigning `val` here, so I switched it to call the setter explicitly.